scripts/simplify_network.py

```diff
@@ -620,6 +766,20 @@ def consense(x):
 
     update_p_nom_max(n)
 
+    n = remove_empty_isolated_nodes(n)
+
+    e_threshold_drop_isolated = cluster_config.get("e_threshold_drop_isolated", False)
+    e_threshold_merge_isolated = cluster_config.get("e_threshold_merge_isolated", False)
+    if e_threshold_drop_isolated:
+        n = drop_isolated_nodes(n, drop_e_threshold=e_threshold_drop_isolated)
```

Comment:
Above I added a suggestion to merge remove_empty isolated nodes and this function.
In that case, drop_isolated_nodes will always be executed by the drop_threshold would be max(0.0, config_threshold).
It is worth noticing that any scalar value, even negative would make empty nodes be dropped, see description above

Reply:
Done :)